webui/react/src/shared/styles/antd.scss

```diff
@@ -217,9 +217,12 @@ html {
     opacity: 1;
     user-select: auto;
   }
-  .ant-table .ant-table-tbody > tr > td[class*=ant-table-cell-fix-] {
+  .ant-table .ant-table-tbody > tr > td.ant-table-cell {
     background-color: var(--theme-stage-weak);
   }
+  .ant-table .ant-table-tbody > tr > td.ant-table-cell.ant-table-column-sort {
+    filter: invert(2%);
```

Comment:
not sure if `filter` is the preferred way to do this, but this at least preserves the visual of:
- highlighted row + sorted column is stronger color than either of the two individually (initial PR did not)

<img width="375" alt="image" src="https://user-images.githubusercontent.com/28383080/181430963-7fd8b296-2206-4d1d-9b82-c36b130f3ac4.png">